UPGRADING.md

```diff
@@ -116,6 +116,22 @@ params do
 end
 ```
 
+#### Coercing to `FalseClass` or `TrueClass` no longer works
+
+Previous Grape versions allowed this, though it wasn't documented:
+
+```ruby
+requires :true_value, type: TrueClass
+requires :bool_value, types: [FalseClass, TrueClass]
+```
+
+These will now crash. Instead you should do:
```

Comment:
Crash? What actually happens?

Reply:
https://github.com/ruby-grape/grape/blob/master/lib/grape/validations/types/dry_type_coercer.rb#L65 <- this line crashes, because `@coercer` is `false` or `true`, rather than an instance of a coercer.

Reply:
I just mean we need to say what users get instead of saying "Crash", ie. explain this to the users. It should say something like this: "The example above will now cause `error bla bla bla`." To work around this, use a `Boolean` and check for its value.

Reply:
I clarified the wording. The exception is actually handled on the next line, so it's not a crash, it's just that the value is never considered valid.